snake.py

```diff
@@ -15,64 +17,87 @@
+# Prints the food
+win.addch(food[0], food[1], '*')
 
-while key != 27:                                                   # While Esc key is not pressed
+# While Esc key is not pressed
+while key != 27:
     win.border(0)
-    win.addstr(0, 2, 'Score : ' + str(score) + ' ')                # Printing 'Score' and
-    win.addstr(0, 27, ' SNAKE ')                                   # 'SNAKE' strings
-    win.timeout(150 - (len(snake)/5 + len(snake)/10)%120)          # Increases the speed of Snake as its length increases
-    
-    prevKey = key                                                  # Previous key pressed
-    event = win.getch()
-    key = key if event == -1 else event 
+    # Printing 'Score' and
+    win.addstr(0, 2, 'Score : ' + str(score) + ' ')
+    # 'SNAKE' strings
+    win.addstr(0, 27, ' SNAKE ')
+    # Increases the speed of Snake as its length increases
+    win.timeout(150 - (len(snake)/5 + len(snake)/10) % 120)
 
+    # Previous key pressed
+    prevKey = key
+    event = win.getch()
+    key = key if event == -1 else event
 
-    if key == ord(' '):                                            # If SPACE BAR is pressed, wait for another
-        key = -1                                                   # one (Pause/Resume)
+    # If SPACE BAR is pressed, wait for another
+    if key == ord(' '):
+        # one (Pause/Resume)
+        key = -1
         while key != ord(' '):
             key = win.getch()
         key = prevKey
         continue
 
-    if key not in [KEY_LEFT, KEY_RIGHT, KEY_UP, KEY_DOWN, 27]:     # If an invalid key is pressed
+    # If an invalid key is pressed
+    if key not in [KEY_LEFT, KEY_RIGHT, KEY_UP, KEY_DOWN, 27]:
         key = prevKey
 
-    # Calculates the new coordinates of the head of the snake. NOTE: len(snake) increases.
+    # Calculates the new coordinates of the head
+    # of the snake. NOTE: len(snake) increases.
     # This is taken care of later at [1].
-    snake.insert(0, [snake[0][0] + (key == KEY_DOWN and 1) + (key == KEY_UP and -1), snake[0][1] + (key == KEY_LEFT and -1) + (key == KEY_RIGHT and 1)])
+    snake.insert(0, [snake[0][0] + (key == KEY_DOWN and 1) +
+                 (key == KEY_UP and -1), snake[0][1] +
+                 (key == KEY_LEFT and -1) + (key == KEY_RIGHT and 1)])
 
     # If snake crosses the boundaries, make it enter from the other side
-    if snake[0][0] == 0: snake[0][0] = 18
-    if snake[0][1] == 0: snake[0][1] = 58
-    if snake[0][0] == 19: snake[0][0] = 1
-    if snake[0][1] == 59: snake[0][1] = 1
+    if snake[0][0] == 0:
+        snake[0][0] = 18
+    if snake[0][1] == 0:
+        snake[0][1] = 58
+    if snake[0][0] == 19:
+        snake[0][0] = 1
+    if snake[0][1] == 59:
+        snake[0][1] = 1
 
     # Exit if snake crosses the boundaries (Uncomment to enable)
-    #if snake[0][0] == 0 or snake[0][0] == 19 or snake[0][1] == 0 or snake[0][1] == 59: break
+    # if snake[0][0] == 0 or snake[0][0] == 19 or
```

Comment:
If you put the if statements inside parentheses, you can use an implied line continuation, where you can break and indent more logically, like 
`# if (snake[0][0] == 0 or snake[0][0] == 19 or
    #         snake[0][1] == 0 or snake[0][1] == 59):
    #     break`

Reply:
Of course, there was no linting error here...  editing this is for overachievers only :laughing: